src/utils/daos_dfs_hdlr.c

```diff
@@ -310,3 +311,41 @@ fs_relink_root_hdlr(struct cmd_args_s *ap)
 {
 	return dfs_relink_root(ap->cont);
 }
+
+int
+fs_chmod_hdlr(struct cmd_args_s *ap)
```

Comment:
We have been trying to (slowly) shift away from these C handler functions, as they make it impossible to print the output as JSON. Better to implement the same logic in the Go code. You can make calls out to the dfs API using CGo. I believe there are some examples of this approach near the bottom of `src/control/cmd/daos/filesystem.go`.

Reply:
There seems to be a mix of things here.  Would it be possible to defer this and I'll file another ticket to fix it?  One issue is it will cause merge conflicts with the patch on google/2.6 (which we needed immediately) so would be a bit easier if I can do it in a separate patch.

Reply:
Okay, that's fair.